.changeset/funny-bananas-compare.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/localized-rich-text-input': patch
+---
+
+Improve `<LocalizedRichTextInput>`:
+- change `onChange` prop handling to make the component work with Formik
+- make the component resettable
```

Comment:
```suggestion
Fix regression of `onChange` to pass `target.name` and `target.id` with the locale suffix.

Furthermore, the reset functionality works correctly for all localized inputs.
```

Reply:
Since `resetValue` function is the new way of resetting the component values, perhaps we could also mention how is it supposed to be used or [link to the release note](https://github.com/commercetools/ui-kit/releases/tag/%40commercetools-uikit%2Frich-text-utils%4015.0.0) where we introduced this for `rich-text-utils` package.